ait-extension/src/js/api/client.ts

```
@@ -0,0 +1,289 @@
+  }
+};
+
+// Ingest API
+export const ingestApi = {
+  async startIngest(directory: string, options: IngestOptions) {
+    try {
+      const response = await apiClient.post('/ingest', { directory, ...options });
+      return response.data;
+    } catch (error: any) {
+      console.error('AIServer: Error starting ingest:', error);
+      if (error.response && error.response.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)